### PR TITLE
Correct branch names for security and logstash repos

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
Closes PR #335 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by hdhalter — APPROVED** [review]

File: _clients/data-prepper/pipelines.md [hunk 1]
```diff
@@ -71,10 +71,14 @@ log-pipeline:
...
 {: .note}
 
-### Trace Analytics pipeline
+### Trace analytics pipeline
 
 The following example demonstrates how to build a pipeline that supports the [Trace Analytics OpenSearch Dashboards plugin]({{site.url}}{{site.baseurl}}/observability-plugin/trace/ta-dashboards/). This pipeline takes data from the OpenTelemetry Collector and uses two other pipelines as sinks. These two separate pipelines index trace and the service map documents for the dashboard plugin.
 
+#### Classic
+
+This pipeline definition will be deprecated in 2.0. Users are recommended to use [Event record type](#event-record-type) pipeline definition.
```
Comment (lizsnyder) [inline _clients/data-prepper/pipelines.md:80]: We recommend using the Event record type pipeline definition.

File: _clients/data-prepper/data-prepper-reference.md [hunk 2]
```diff
@@ -116,13 +142,19 @@ Prior to Data Prepper 1.3, Processors were named Preppers. Starting in Data Prep
...
-root_span_flush_delay | No | Integer | Represents the time interval in seconds to flush all the root spans in the processor together with their descendants. Default is 30.
 trace_flush_interval | No | Integer | Represents the time interval in seconds to flush all the descendant spans without any root span. Default is 180.
 
+### otel_trace_raw
+
+This processor is a Data Prepper event record type compatible version of `otel_trace_raw_prepper` that fills in trace group related fields into all incoming Data Prepper span records. It requires `record_type` to be set as `event` in `otel_trace_source`.
+
+Option | Required | Type | Description
+:--- | :--- | :--- | :---
+trace_flush_interval | No | Integer | Represents the time interval in seconds to flush all the descendant spans without any root span. Default is 180.
```
Comment (lizsnyder) [inline _clients/data-prepper/data-prepper-reference.md:157]: time interval, in seconds, to

File: _clients/data-prepper/data-prepper-reference.md [hunk 3]
```diff
@@ -65,7 +68,30 @@ request_timeout | No | Integer | The request timeout in millis. Default is `10_0
...
+thread_count | No | Integer | The number of threads to keep in the ScheduledThreadPool. Default is `200`.
+max_connection_count | No | Integer | The maximum allowed number of open connections. Default is `500`.
+ssl | No | Boolean | Enables connections to the OTel source port over TLS/SSL. Defaults to `true`.
+sslKeyCertChainFile | Conditionally | String | File-system path or AWS S3 path to the security certificate (e.g. `"config/demo-data-prepper.crt"` or `"s3://my-secrets-bucket/demo-data-prepper.crt"`). Required if `ssl` is set to `true`.
+sslKeyFile | Conditionally | String | File-system path or AWS S3 path to the security key (e.g. `"config/demo-data-prepper.key"` or `"s3://my-secrets-bucket/demo-data-prepper.key"`). Required if `ssl` is set to `true`.
+useAcmCertForSSL | No | Boolean | Whether to enable TLS/SSL using certificate and private key from AWS Certificate Manager (ACM). Default is `false`.
+acmCertificateArn | Conditionally | String | Represents the ACM certificate ARN. ACM certificate take preference over S3 or local file system certificates. Required if `useAcmCertForSSL` is set to `true`.
```
Comment (lizsnyder) [inline _clients/data-prepper/data-prepper-reference.md:90]: ACM certificates

File: _clients/data-prepper/data-prepper-reference.md [hunk 4]
```diff
@@ -65,7 +68,30 @@ request_timeout | No | Integer | The request timeout in millis. Default is `10_0
...
+Source for the OpenTelemetry Collector for collecting metric data.
+
+Option | Required | Type | Description
+:--- | :--- | :--- | :---
+port | No | Integer | The port OTel metrics source is running on. Default is `21891`.
+request_timeout | No | Integer | The request timeout in milliseconds. Default is `10_000`.
+health_check_service | No | Boolean | Enables a gRPC health check service under `grpc.health.v1/Health/Check`. Default is `false`.
+proto_reflection_service | No | Boolean | Enables a reflection service for Protobuf services (see [gRPC reflection](https://github.com/grpc/grpc/blob/master/doc/server-reflection.md) and [gRPC Server Reflection Tutorial](https://github.com/grpc/grpc-java/blob/master/documentation/server-reflection-tutorial.md) docs). Default is `false`.
+unframed_requests | No | Boolean | Enable requests not framed using the gRPC wire protocol.
+thread_count | No | Integer | The number of threads to keep in the ScheduledThreadPool. Default is `200`.
```
Comment (lizsnyder) [inline _clients/data-prepper/data-prepper-reference.md:84]: `ScheduledThreadPool`

**Review by lizsnyder — APPROVED** [review]

**Comment by Naarcha-AWS** [comment]
@natebower: This as well!

**Review by alicejw-aws — APPROVED** [review]

**Pull request merged**